inc/based_config.php

```diff
@@ -101,14 +101,17 @@
         'GLPI_MARKETPLACE_MANUAL_DOWNLOADS' => true, // propose manual download link of plugins which cannot be installed/updated by marketplace
         'GLPI_USER_AGENT_EXTRA_COMMENTS'    => '', // Extra comment to add to GLPI User-Agent
 
-      // SQL compatibility
+        // SQL compatibility
         'GLPI_DISABLE_ONLY_FULL_GROUP_BY_SQL_MODE' => '1', // '1' to disable ONLY_FULL_GROUP_BY 'sql_mode'
 
-      // Other constants
+        // Other constants
         'GLPI_AJAX_DASHBOARD'         => '1', // 1 for "multi ajax mode" 0 for "single ajax mode" (see Glpi\Dashboard\Grid::getCards)
         'GLPI_CALDAV_IMPORT_STATE'    => 0, // external events created from a caldav client will take this state by default (0 = Planning::INFO)
         'GLPI_DEMO_MODE'              => '0',
         'GLPI_CENTRAL_WARNINGS'       => '1', // display (1), or not (0), warnings on GLPI Central page
+
+        // Documentations URLs constants
+        'GLPI_DOCURL_CRONTASKS' => 'https://links.glpi-project.org/doc-crontasks',
```

Comment:
Having dedicated documentation link for each feature can help a lot, but we should not have to declare a constant for each of them. Maybe constant should contains only root URL.
```suggestion
        'GLPI_DOCUMENTATION_ROOT_URL' => 'https://links.glpi-project.org',
```